src/pages/signin/PasswordForm.js

```diff
@@ -32,17 +33,21 @@ const propTypes = {
         loading: PropTypes.bool,
     }),
 
+    /** Whether the page is visible. */
+    visible: PropTypes.bool,
+
     ...withLocalizePropTypes,
 };
 
 const defaultProps = {
     account: {},
+    visible: false,
 };
 
 class PasswordForm extends React.Component {
     constructor(props) {
         super(props);
-
+        this.input = null;
```

Comment:
Please remove all refs set as null in constructor.
[Source](https://github.com/Expensify/App/pull/2143)

Reply:
Maybe it's about adding them. :smile:

Reply:
No no, the PR was reverted.
And we agreed to clean it up! [Slack ref](https://expensify.slack.com/archives/C01GTK53T8Q/p1635178933076400?thread_ts=1634850853.025000&cid=C01GTK53T8Q)

Reply:
Thanks. I am still not sure but I will check the styleGuide before doing it or maybe wait for the review to combine changes into one.

Reply:
+1